Kajabi/sage-lib
# Docs: Sidebar updates

## Description
This PR refactors the docs sidebar to use the `SageSidebar` item levels for the heading and the nav items in order to get the active item to highlight. In addition, it adds a script to scroll the active nav item into view.

## Screenshots
|  Before  |  After  |
|--------|--------|
|<img width="241" alt="Before" src="https://user-images.githubusercontent.com/7331038/125115143-ed9c9b80-e0b8-11eb-817d-7aedebd6fdf3.png">|<img width="241" alt="After" src="https://user-images.githubusercontent.com/7331038/125115138-ed040500-e0b8-11eb-8bc1-9791bb7b9427.png">|

## Testing in `sage-lib`
1. Visit http://0.0.0.0:4000/pages/component/pagination and ensure that the sidebar matches the screenshot
2. Visit other component pages and ensure that the selected item remains in the center.


## Testing in `kajabi-products`
N/A

## Related
N/A
